build/azure-pipelines/config/tsaoptions.json

```diff
@@ -1,12 +1,21 @@
 {
-	"instanceUrl": "https://msazure.visualstudio.com/defaultcollection",
-	"projectName": "One",
-	"areaPath": "One\\VSCode\\Client",
-	"iterationPath": "One",
+	"codebaseName": "devdiv_vscode-client",
+	"ppe": false,
 	"notificationAliases": [
-		"sbatten@microsoft.com"
+			"sbatten@microsoft.com"
```

Comment:
I know this line didn't change, but is `sbatten@microsoft.com` correct given that below I see `REDMOND\\stbatt`?

Reply:
yep, this is right, thanks for checking though